kart/conflicts.py

```diff
@@ -216,9 +216,20 @@ def item_to_text(key, value, path, level):
 def conflicts_json_as_geojson(json_obj):
     """Converts the JSON output of list_conflicts to geojson."""
     features = []
+    meta_item_key = ""
```

Comment:
If you have an optional string, prefer to initialise it to None (or if there could be more than one, have an empty collection of strings). Using an empty string is more ambiguous - it might be representing the NULL value, as in, we don't have a string, or it might be a valid value for the string - it depends on the context. So, prefer to use None if you can.

Reply:
Sure, will keep this in mind.